package.json

```diff
@@ -27,6 +27,7 @@
     "@rjsf/core": "^3.0.0",
     "@sage-bionetworks/rjsf-core": "^3.1.3",
     "@upsetjs/react": "^1.6.2",
+    "animate.css": "^4.1.1",
```

Comment:
Adding animate.css to replicate the fadeInUp/fadeOutDown animations we use for toasts in SWC. Just need to make sure this doesn't cause conflicts with animations in SWC.